sbpy/photometry/core.py

```diff
@@ -9,16 +9,31 @@
 __all__ = ['DiskIntegratedPhaseFunc', 'LinearPhaseFunc',
            'InvalidPhaseFunctionWarning']
 
+__doctest_requires__ = {
+    ("DiskIntegratedPhaseFunc",
+     "DiskIntegratedPhaseFunc._phase_integral",
+     "DiskIntegratedPhaseFunc.from_obs",
```

Comment:
I don't think `.from_obs` method uses `scipy`.  Also, `LinearPhaseFunc._phase_integral` is inherited from `DiskIntegratedPhaseFunc`.  Does it need to be included here?

Reply:
This skips the doctest in from_obs's docstring, which does need scipy for the model fitting.